core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -111,7 +111,7 @@ public final class S3RestServiceHandler {
   /* Object is after bucket in the URL path */
   public static final String OBJECT_PARAM = "{bucket}/{object:.+}";
 
-  private final FileSystem mFileSystem;
+  private final FileSystem mMetaFS;
```

Comment:
In all the S3 API classes, there was a refactor to use two separate `FileSystem`s. One which is generated under the scope of a given user per request via the "Authorization" header, and this one which is maintained at the top-level and is occasionally used for Alluxio metadata operations.